server/src/pages/delete-account/server.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+const reactrender = require("../../reactrender");
+const { Shot } = require("../../servershot");
+const { addDeviceActivity } = require("../../users");
+
+let app = express();
+
+exports.app = app;
+
+app.get("/", function (req, res) {
```

Comment:
I think this was copy pasted from the shot index? I don't think / is needed in delete-account